File: CrossFiles/TypeScript/Class1.ts
import { exec } from 'child_process';

export class Class1 {
    private input: string;

    constructor(input: string) {
        this.input = input;
    }

    public process(): void {
        console.log(`Class1 processing: ${this.input}`);
        // Potentially unsafe operation
        exec(`echo ${this.input}`, (error, stdout, stderr) => {
            if (error) {
                console.error(`exec error: ${error}`);
                return;
            }
            console.log(`stdout: ${stdout}`);
            console.error(`stderr: ${stderr}`);
        });
    }
}

File: CrossFiles/Go/Class1.go
package main

import (
	"fmt"
	"os/exec"
)

type Class1 struct {
	input string
}

func NewClass1(input string) *Class1 {
	return &Class1{input: input}
}

func (c *Class1) Process() {
	fmt.Printf("Class1 processing: %s\n", c.input)
	// Potentially unsafe operation
	cmd := exec.Command("echo", c.input)
	output, err := cmd.Output()
	if err != nil {
		fmt.Printf("Error executing command: %v\n", err)
		return
	}
	fmt.Printf("Command output: %s", output)
}
